### PR TITLE
a1200 converter minor changes

### PR DESCRIPTION
**Pull request by 8bits4ever**
-RESET key implementation on layer [1]
-README typo fix
-Info.json formatting for easy reading

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/converter/a1200/info.json [hunk 1]
```diff
@@ -4,13 +4,110 @@
...
-            		   {"label":"Shift", "x":0, "y":4.5, "w":1.25}, {"label":"|", "x":1.25, "y":4.5}, {"label":"Z", "x":2.25, "y":4.5}, {"label":"X", "x":3.25, "y":4.5}, {"label":"C", "x":4.25, "y":4.5}, {"label":"V", "x":5.25, "y":4.5}, {"label":"B", "x":6.25, "y":4.5}, {"label":"N", "x":7.25, "y":4.5}, {"label":"M", "x":8.25, "y":4.5}, {"label":"<", "x":9.25, "y":4.5}, {"label":">", "x":10.25, "y":4.5}, {"label":"?", "x":11.25, "y":4.5}, {"label":"Shift", "x":12.25, "y":4.5, "w":2.75}, {"label":"\u2190", "x":15.25, "y":4.5}, {"label":"\u2193", "x":16.25, "y":4.5}, {"label":"\u2192", "x":17.25, "y":4.5}, {"label":"1", "x":18.5, "y":4.5}, {"label":"2", "x":19.5, "y":4.5}, {"label":"3", "x":20.5, "y":4.5}, {"label":"Enter", "x":21.5, "y":4.5, "h":2}, 
-            		   {"label":"alt", "x":1, "y":5.5, "w":1.5}, {"label":"Amiga", "x":2.5, "y":5.5, "w":1.5}, {"x":4, "y":5.5, "w":7}, {"label":"Amiga", "x":11, "y":5.5, "w":1.5}, {"label":"Alt", "x":12.5, "y":5.5, "w":1.5}, {"label":"0", "x":18.5, "y":5.5, "w":2}, {"label":".", "x":20.5, "y":5.5}]
+        "LAYOUT_default": {
+            "key_count": 96,
```
Comment (fauxpark) [inline keyboards/converter/a1200/info.json:8]:
```suggestion
```

File: keyboards/converter/a1200/keymaps/default/keymap.c [hunk 2]
```diff
@@ -30,7 +30,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
      KC_ESC,    KC_F1, KC_F2, KC_F3, KC_F4, KC_F5,      KC_F6, KC_F7, KC_F8, KC_F11, KC_F12,
```
Comment (fauxpark) [inline keyboards/converter/a1200/keymaps/default/keymap.c:30]: You might want to set the keys that are the same on this layer to `_______` (aka `KC_TRNS`) for clarity and readability.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/converter/a1200/keymaps/default/keymap.c (hunk 2 above)
Comment (8bits4ever) [reply keyboards/converter/a1200/keymaps/default/keymap.c:30]: Yes, I'll do that.

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by 8bits4ever** [comment]
> Thanks!

You are welcome!

I don't know if this is the right channel to ask, but where is the best place to put a variant of this keyboard that compiles for a different micro? Currently this is for the AT90USB128x but I have a version for the ATMEGA32U4 (cheaper micro, better availability)...

**Comment by noroadsleft** [comment]
@8bits4ever,

0. Create a `teensy2` directory under `keyboards/converter/a1200/`, and move your `config.h` and `rules.mk` files into it.
1. Create a `promicro` directory under  `keyboards/converter/a1200/`, and create `config.h` and `rules.mk` files there that would be appropriate for it (assuming you're going for a Pro Micro).

**Comment by 8bits4ever** [comment]
@noroadleft

Thanks. Only config.h and rules.mk files should be moved? The rest of the files remain at keyboards/converter/a1200/?

**Comment by noroadsleft** [comment]
> Thanks. Only config.h and rules.mk files should be moved? The rest of the files remain at keyboards/converter/a1200/?

Any code in `keyboards/converter/a1200/` will be shared among all versions of the board; the subdirectories only need the code specific to their version.

> Actually doing a custom PCB for the Atmega32U4. The promicro has not enough pins available.

In that case, name the folder `32u4`, but the other guidance applies.

Also I misread your readme, and that `teensy2` directory should be named `teensy2pp` (to specify it's the 2.0++ rather than the standard Teensy 2.0).

**Comment by 8bits4ever** [comment]
Excellent. Thanks again

**Pull request merged**